src/object/C4ObjectList.cpp

```diff
@@ -574,42 +767,74 @@ void C4ObjectList::Sort()
 	do
 	{
 		fSorted = true;
-		for (cLnk=First; cLnk && cLnk->Next; cLnk=cLnk->Next)
+		for (cLnk = First; cLnk && cLnk->Next; cLnk = cLnk->Next)
+		{
 			if (cLnk->Obj->id > cLnk->Next->Obj->id)
 			{
 				RemoveLink(cLnk);
-				InsertLink(cLnk,cLnk->Next);
+				InsertLink(cLnk, cLnk->Next);
 				fSorted = false;
 				break;
 			}
+		}
 	}
 	while (!fSorted);
 }
 
 void C4ObjectList::RemoveLink(C4ObjectLink *pLnk)
 {
-	if (pLnk->Prev) pLnk->Prev->Next=pLnk->Next; else First=pLnk->Next;
-	if (pLnk->Next) pLnk->Next->Prev=pLnk->Prev; else Last=pLnk->Prev;
+	if (pLnk->Prev)
+	{
```

Comment:
I actually like the short variant better here because it shows the symmetry, but whatever.

In any case, it might be worth it to standardize the normal "link" variable name in this file. `cLnk` seems to be the most common one (whatever the c stands for), but there's also `clnk` (which is very confusing because it would stand for "Clonk" everywhere else) and `pLnk` (here).

Reply:
I'll change it back to the old notation then.